keys.js

```diff
@@ -0,0 +1,16 @@
+// console.log('this is loaded');
+
+var keys = {
+  twitterKeys: {
+    consumer_key: 'LKZ1mRv94YzKIdkI2Ss7C12gJ',
+    consumer_secret: 'bH1QRcmBwNJjo8akpyvostnxls0uHyiEaLLXWbWoAbDYzXmlFo',
+    access_token_key: '214569195-mWSJf9NfA37as7qC2D824oBBex0idQgzvI0L5Uq9',
+    access_token_secret: 'PWxsy26wE909JGpQzvhAqmXbrQP2G9eRVmMjd90ZEyDHH'
+  },
+  spotifyKeys: {
+    client_ID: 'cc8b067f81a24ee2b49e96cecf496480',
+    client_Secret: 'e133dca456fc4b0c86be3612454c2357'
+  },
+}
+
+module.exports = keys;
```

Comment:
I would instead, export the instances of your modules instead of just the credentials.